whois-client/src/main/java/net/ripe/db/whois/query/QueryMessages.java

```diff
@@ -291,6 +291,12 @@ public static Message accessDeniedPermanently(final String accountingId) {
                 accountingId);
     }
 
+    public static Message accessDeniedForAbuse(final String accountingId) {
+        return new QueryMessage(Type.ERROR, ""
+                + "Your host %s has been permanently blocked due to suspected abusive behaviour. Please contact support for further assistance.",
```

Comment:
Include an email address `<ripe-dbm@ripe.net>` after `support`.

Reply:
We also should review the text and maybe replace `suspected abusive` with something more neutral, as they may be blocked inadvertently

Reply:
Decided to use "Your host %s has been blocked. Please contact support <ripe-dbm@ripe.net> for further assistance."